src/test/functional/features/postcodes.feature

```diff
@@ -9,30 +9,36 @@ Feature: Postcodes
     Then I can view the courts or tribunals in a list format
     When I click edit next to court with "aldershot-justice-centre"
     Then I am redirected to the Edit Court page for the chosen court
-    When I hover over opening hours nav element
+    And I hover over types nav element
     When I click the types tab
     And I will make sure County court type is selected
     When I click the postcodes tab
 
   Scenario: Adding and deleting valid postcodes
-    When I add new postcodes "bd7 2qb,bd2,bd4,BD7"
+    When I will make sure to delete the existing postcodes
+    When I add new postcodes "bd7 2qb,bd3,BD7"
     Then I click the add postcode button
-    Then I click the select all
-    When I click the delete all selected button
     Then A green message is displayed for the postcodes "Postcodes updated"
-    When I add new postcodes "bd7 2qb,bd2,bd4,BD7"
-    Then I click the add postcode button
+    When I will make sure to delete the existing postcodes
     Then A green message is displayed for the postcodes "Postcodes updated"
 
   Scenario: Adding invalid and duplicate postcode
     When I add new postcodes "bdx,123,bd1"
     Then I click the add postcode button
     Then The error message display for the postcodes "A problem has occurred (your changes have not been saved). The following postcodes are invalid: bdx,123"
-    When I add new postcodes "bd7 2qb"
+    When I add new postcodes "bd1"
+    Then I click the add postcode button
+    When I add new postcodes "bd1"
     Then I click the add postcode button
-    Then The error message display for the postcodes "One or more postcodes provided already exist: BD72QB"
+    Then The error message display for the postcodes "One or more postcodes provided already exist: BD1"
 
   Scenario Outline: Moving postcodes from the source court to the destination court
+    When I will make sure to delete the existing postcodes
+    When I add new postcodes "bd10,bd2,bd4"
+    Then I click the add postcode button
+    Then A green message is displayed for the postcodes "Postcodes updated"
+    Then I will make sure to delete the existing postcodes for the court "<view_court_slug>"
+    Then I went back to source court "aldershot-justice-centre"
```

Comment:
This step could perhaps be named to describe more fully what it's doing i.e. opening the different court for editing of postcodes.